bundles/org.openhab.persistence.dynamodb/README.md

```diff
@@ -48,7 +48,7 @@ Please also note possible [Free Tier](https://aws.amazon.com/free/) benefits.
 **Note:** The below policy assumes that `eu-west-1` region is used, the new table schema is used, and the default table name of `openhab` is used.
 Modify the policy accordingly if needed.
 
-**Note 2:** As a more simple alternative, one can use pre-existing policy `AmazonDynamoDBFullAccess`, although the policy grants the openHAB user wider-than-necessary permissions.
+**Note2:** The Number in arn:aws:dynamodb:eu-west-1:084669220525:table/openhab must be replaced by the unique ID of your own table to grant access for your new db user. To get this ID there are two Options:The first one is that you create the table manually in the AWS Console. Once the table is created you could click on the new table and could see your unique Amazon Resource Name starting with arn: which you must replace in the JSON script.  The other is that you give your new database user the existing IAM policy `AmazonDynamoDBFullAccess` to grant him the right to create the new table. After that you could revoke the `AmazonDynamoDBFullAccess` from the openhab user for security reasons.
```

Comment:
I would not recommend creating the table manually as you would need to carefully configure all the things by hand. (Time to live and whatnot) incorrect config will lead to all sort of issues.

The binding configures these things only when the binding is creating the table.

Can we not change the policy on this page such that it would allow creating the table, similar how AmazonDynamoDBFullAccess allows creating a table? I might have some time to look this during the next week or two.